crypto/kyber/kem_kyber.h

```diff
@@ -7,7 +7,7 @@
 #include <openssl/evp.h>
```

Comment:
Missing copyright, is this our file or kyber's originally?

Reply:
added. it's our file